### PR TITLE
ZK-4472: Using fileupload in MVVM Client-side binding

### PR DESCRIPTION
**Pull request by DevChu**

### CODE REVIEW

File: zkbind/src/archive/web/js/zkbind/Binder.js [hunk 1]
```diff
@@ -249,6 +249,24 @@ zkbind.Binder = zk.$extends(zk.Object, {
...
 			ac[i].apply(this, [args]);
+	},
+	upload: function (cmd, file, opts, timeout) {
+		var formData = new FormData(),
+			xhr = new XMLHttpRequest(),
+			wgt = this.$view,
+			sid = wgt._sid ? ++wgt._sid : (wgt._sid = 0);
+
+		formData.append('file', file);
+		xhr.onload = function (e) {
```
Comment (DevChu) [inline zkbind/src/archive/web/js/zkbind/Binder.js]: delete e

File: zkbind/src/archive/web/js/zkbind/Binder.js [hunk 2]
```diff
@@ -249,6 +249,24 @@ zkbind.Binder = zk.$extends(zk.Object, {
 		var ac = this._aftercmd[cmd];
 		for (var i = 0, j = ac ? ac.length : 0; i < j; i++)
 			ac[i].apply(this, [args]);
+	},
+	upload: function (cmd, file, opts, timeout) {
```
Comment (DevChu) [inline zkbind/src/archive/web/js/zkbind/Binder.js]: remove opts, timeout

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**